javascript/react/src/components/ArqueroDetailsList/renderers/StatsColumnHeader.hooks.ts

```diff
@@ -12,8 +12,8 @@ import { pretty } from './StatsColumnHeader.constants.js'
 
 export function useTooltip(stats?: FieldMetadata): string {
 	return useMemo(() => {
-		const { bins, categories, ...nobins } = stats || EMPTY_OBJECT
-		return Object.entries(nobins).reduce((acc, cur, idx) => {
+		const { bins, categories, ...args } = stats || EMPTY_OBJECT
```

Comment:
I don't think it does - `nobins` is not a meaningful input name, but it's what we assign leftovers to.

However, I'll revert this since the compound-word setting will allow this.